test/Streamly/Test/Data/Unbox.hs

```diff
@@ -70,10 +166,21 @@ tests = do
         $ checkSizeOf (Proxy :: Proxy SumOfProducts) 25
     prop "checkSizeOf NestedSOP" $ checkSizeOf (Proxy :: Proxy NestedSOP) 26
 
+testsManual :: Spec
+testsManual = do
+    prop "Int type"  $ testInt (WInt 5)
+    prop "Int8 type" $ testInt8 (WInt8 127)
+    prop "Int16 type" $ testInt16 (WInt16 444)
+    prop "Int32 type" $ testInt32 (WInt32 7777)
+    prop "Float type" $ testFloat (WFloat 33.556)
+    prop "Double type" $ testDouble (WDouble 336.588856)
+    prop "Word32 type" $ testWord32 (WWord32 4294967295)
```

Comment:
We discussed that we will write tests only for non-primitive types. These are all primitive types, there is not much point writing tests for these, they are only testing what the `Single` test is doing, it is testing a single wrapper type.

We need to test only those instances which are not primitive i.e. those that can be derived by generic deriving using the primitive type instances.